styles.css

```diff
@@ -62,9 +62,283 @@ body {
 .social-media-icons {
   height: 30px;
   display: flex;
+  margin: 8px;
 }
 
 .social-media-icons a {
   text-decoration: none;
-  margin: 8px;
+  align-items: center;
+  margin: 6px;
+}
+
+.social-media-icons img {
+  padding: 6px;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 2650px;
+  padding: 114px 24px;
+}
+
```

Comment:
- [ ] As requested by the project requirements, the container of the projects should have a `display: grid;` property. Kindly change this